src/csharp/Microsoft.Spark.Worker/Processor/BroadcastVariableProcessor.cs

```diff
@@ -31,19 +34,19 @@ internal BroadcastVariables Process(Stream stream)
                 broadcastVars.DecryptionServerNeeded = SerDe.ReadBool(stream);
             }
 
-            // Note that broadcast variables are currently ignored.
-            // Thus, just read the info from stream without handling them.
             int numBroadcastVariables = Math.Max(SerDe.ReadInt32(stream), 0);
```

Comment:
Do we need a `numBroadcastVariables` variable now?  Could we just assign directly to `broadcastVars.Count` ?

Reply:
We could, will make that change.